js-exercises/update-object/updateObject.test.js

```diff
@@ -0,0 +1,23 @@
+import { updateObject } from './updateObject';
+
```

Comment:
Nice work on writing tests. One more test you could add about checking whether the returned array is a new Array or not.